pkg/config/config.go

```diff
@@ -202,6 +202,10 @@ type PluginConfig struct {
 	// DisableProcMount disables Kubernetes ProcMount support. This MUST be set to `true`
 	// when using containerd with Kubernetes <=1.11.
 	DisableProcMount bool `toml:"disable_proc_mount" json:"disableProcMount"`
+	// DefaultOCIHooks (optional) is a path to a json file that specifies a
+	// default OCI spec Hooks struct. ** Note: The any hooks set by default can be
+	// overriden if/when the hooks are set via the CRI.
```

Comment:
`s/The any/any/`?

Can the CRI remove a default hook on `poststop` but not touch another default hook on `prestart`? I would prefer not, so that the hooks remain consistent and that any resources allocated in `poststart` are correctly released in `poststop`.